File: src/main/java/com/movie4u/sep/models/People.java
package com.movie4u.sep.models;

import lombok.ToString;

@ToString
public class People {
    private int id;
    private String name;
    private Long birth;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Long getBirth() {
        return birth;
    }

    public void setBirth(Long birth) {
        this.birth = birth;
    }
}

File: SEP6/Models/People.cs
namespace SEP6.Models
{
    public class People
    {
        public int Id { get; set; }
        public string name { get; set; }
        public int birthYear { get; set; }
        public List<Directors> directors { get; set; }
        public List<Stars> stars { get; set; }
    }
}
